### ISSUE TITLE
IcCheckbox automated accessibility failure

### ISSUE THREAD

**Issue by GCHQ-Developer-718** [mi6/ic-ui-kit#3879]
### Summary of the bug 
A user has reported that the IcCheckbox has caused an automated accessibility failure with Accessibility Insights.

The tool identified the following error with the checkmark svg:

> Image function: every image must be coded as either meaningful or decorative
> 
> Path: ic-checkbox,.checkmark
> Snippet: `<svg class="checkmark" viewbox= ... ><title>checkmark icon</title><path ...></path></svg>`
> Coded as: decorative
> ARIA role: no value
> Accessible name: checkmark icon

This could be a false positive as the image has an accessible name so I'm not sure why the tool thinks it is decorative.
